azerothcore/azerothcore-wotlk
# fix(Scripts/EmeraldDragons): Lethon should use shadow bolt whirl

(Thanks to Nefertumm for some help :) )

## Changes Proposed:
-  Added aura script

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/11890

## SOURCE:

Cmangos https://github.com/cmangos/mangos-classic/commit/26edc21e23df03629872f45fa362a1ccc9672ea0

## Tests Performed:

- Tested


## How to Test the Changes:


1. Go to Lethon with 2 or 3 chars
2. Engage
3. notice that they will shoot shadowbolts, but only 1 side at a time and rotating (So left right left right)

## Known Issues and TODO List:



## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
